integration_tests/CMakeLists.txt

```
@@ -1793,3 +1793,4 @@ RUN(NAME exit_02 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
 RUN(NAME struct_type_01 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
 
 RUN(NAME func_parameter_type_02 LABELS gfortran) # function passed in other argument of function
+RUN(NAME string_negative_start_index2 LABELS gfortran)
```

Test with LFortran too:
```suggestion
RUN(NAME string_negative_start_index2 LABELS gfortran llvm)
```